### DISCUSSION TITLE
Can we do this to diffuser too?

### DISCUSSION THREAD

**Discussion by osi1880vr** [Doggettx/stable-diffusion#14] [Q&A] [answered]
It works so well and impressive in the attention.py that I wonder if you could do the same magic to diffuser?
I found similar code there in their version of attention.py but when I change it only parts work.

We try to get the txt2vid working on smaler cards, but the diffuser library is somehow in the way. Could you have a look
into this? Actualy it is using diffuser 0.2.4 but I guess a way forward to 0.3.0 would be good. But maybe its enough once we understand what needs to be changed where to have the whole process working with your mempory saving magic

Many thanks in advance for just reading this ;)

**Comment by Doggettx** [comment, marked as answer]
They already seem to have implemented it as well, without the memory detection part but the slicing is in there in the _sliced_attention method of attention.py
